src/lib/services/playground/index.ts

```diff
@@ -205,6 +205,15 @@ export const updateRequest = async ({
     };
   }
 
+  await prisma.user.update({
+    where: {
+      id: requesterId,
+    },
+    data: {
+      name: params.name,
+    },
+  });
```

Comment:
Do we want to this behind a condition to only run this update if the `name` changed?? Wasn't sure if it's worth doing the query to get the user.